kuma/landing/views.py

```diff
@@ -118,7 +117,6 @@ def promote_buttons(request):
 '''
 
 
-@shared_cache_control
 def robots_txt(request):
```

Comment:
Maybe replace the decorator with a comment indicating that it is cached, but the decorators are applied in the various urls.py files?

Reply:
My thought was that it seemed overkill to define separate views for the `beta` endpoints just so we could decorate them differently, but I'm fine with doing that way as well.

Reply:
You do as you want. For me, it's not a blocker, but just a shared feeling/opinion.